### PR TITLE
HPCC-19866 Log Visualization displays wrong information if not installed

### PR DESCRIPTION
**Pull request by kunalaswani**
Log Visualization displays wrong message if Kibana is not configured.

Signed-off-by: Aswani, Kunal <kunal.aswani@lexisnexis.com>

## Type of change:
- [x] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [x] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [x] The commit message title makes sense in a changelog, by itself.
  - [x] The commit is signed.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [x] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [ ] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [ ] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [x] This is a user interface / front-end modification
  - [x] I have tested my changes in multiple modern browsers
  - [x] The component(s) render as expected

### CODE REVIEW

**Comment by kunalaswani** [comment]
@GordonSmith please review

File: esp/src/eclwatch/HPCCPlatformOpsWidget.js [hunk 1]
```diff
@@ -80,14 +80,17 @@ define([
                         WsELK.GetConfigDetails({
                             request: {}
                         }).then(function (response) {
-                            if (lang.exists("GetConfigDetailsResponse", response)) {
+                            if (response.GetConfigDetailsResponse.IntegrateKibana === true) {
```
Comment (GordonSmith) [inline esp/src/eclwatch/HPCCPlatformOpsWidget.js]: Not sure I would get rid of the lang exists test?
```
if (lang.exists("GetConfigDetailsResponse.IntegrateKibana", response) && response.GetConfigDetailsResponse.IntegrateKibana === true)
```

File: esp/src/eclwatch/HPCCPlatformOpsWidget.js [hunk 2]
```diff
@@ -80,14 +80,17 @@ define([
...
                                 var elk = response.GetConfigDetailsResponse;
                                 currSel.set("content", dojo.create("iframe", {
                                     src: dojoConfig.urlInfo.pathname + "?Widget=IFrameWidget&src=" + encodeURIComponent(elk.KibanaAddress + ":" + elk.KibanaPort + elk.KibanaEntryPointURI),
                                     style: "border: 0; width: 100%; height: 100%"
                                 }));
                             } else {
-                                //load LogVisualizationWidget and show status there
+                                currSel.set("content", dojo.create ("div",{
+                                    innerHTML: "<p>Log Visualization is not configured, please check your confriguration manager settings.</p> <br> <a href = 'https://hpccsystems.com/blog/ELK_visualizations'>Learn More</a>",
```
Comment (GordonSmith) [inline esp/src/eclwatch/HPCCPlatformOpsWidget.js]: Needs to be moved into translation files - but keep the <p> and <br> (all html formatting) here.

**Review by GordonSmith — CHANGES_REQUESTED** [review]

**Comment by richardkchapman** [comment]
@kunalaswani

**Comment by kunalaswani** [comment]
@GordonSmith suggested changes made, and tested.

**Review by GordonSmith — APPROVED** [review]
Looks good.

**Pull request merged**